Code-the-Dream-School/dd-prac-team1-front
# Dev/remove consolelogs (PRA-201)

## One Line Description
remove all console.log
## Requirements

_What this code should do_

## Notes
fixed last time bugs: 
1. permanent toast ("forgot password page") 
2. hiding behind toast existing ingredient ("shopping list page")
3. notions that render during search ("saved recipe page")
4. the page height ("contact page")
_Any additional info_

## Test Steps
1. check "forgot password page" and send email. press home page button. the toast should disappear
2. check "shopping list page" when you have ingredients there. add new ingredient,  check that you will see existing ingredient on the center with temporary background
3. check "saved recipe page" try to search something and click on any category (except all category) button. Check how notifications are rendering
4. press button "contact" located in footer. Make the page narrow, check how it looks like.

_Instructions on how to test the changes_

## UI/UX Outcome
before: 
<img width="1152" alt="Pasted Graphic 19" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/830c7fa6-47d0-410c-92ea-bd8353625eb9">

after: 
<img width="1123" alt="Pasted Graphic 20" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/86063cfd-d598-4921-b31d-72e47bf94add">


before:
<img width="1346" alt="Pasted Graphic 21" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/023c19ba-6708-4126-ba1e-08cd2a8c1f61">

after: 
<img width="1323" alt="Pasted Graphic 22" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/1753251c-582e-4e17-ba63-fa877ddcb6a5">


before:
<img width="1176" alt="Pasted Graphic 23" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/57df0d54-5e91-4507-931d-b888bb17234a">

after:
<img width="1176" alt="Pasted Graphic 24" src="https://github.com/Code-the-Dream-School/dd-prac-team1-front/assets/106414157/8cffb5c6-870a-4f3f-b418-b0b5b347c606">


_Attach a screenshot here_

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed
